### PR TITLE
New version release 2.0.6

### PR DESCRIPTION
**Pull request by zhongjiajie**
do not merge until https://github.com/apache/dolphinscheduler-website/pull/808 and https://github.com/apache/dolphinscheduler-website/pull/809 merged

### CODE REVIEW

**Review by kezhenxu94 — DISMISSED** [review]

**Review by SbloodyS — DISMISSED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -100,8 +100,8 @@ body:
       options:
         - dev
         - 3.0.0-beta-2
+        - 2.0.6
         - 2.0.5
-        - 2.0.3
         - 1.3.9
         - 1.3.8
```
Comment (SbloodyS) [inline .github/ISSUE_TEMPLATE/bug-report.yml]: Should we remove 1.3.8 and 1.3.9?

Comment (zhongjiajie) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: agrees with that, will change it

Comment (zhongjiajie) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: agrees with that, will change it

**Review by SbloodyS — APPROVED** [review]

**Comment by zhongjiajie** [comment]
FYI @SbloodyS label bot run failed

**Comment by SbloodyS** [comment]
> FYI @SbloodyS label bot run failed

Sad...

**Pull request merged**